src/client/sandbox/node/window.js

```diff
@@ -166,8 +166,12 @@ export default class WindowSandbox extends SandboxBase {
         }
 
         if (window.EventSource) {
-            window.EventSource           = url => new nativeMethods.EventSource(getProxyUrl(url));
-            window.EventSource.prototype = nativeMethods.EventSource.prototype;
+            window.EventSource            = url => new nativeMethods.EventSource(
```

Comment:
`window.EventSource` has two arguments in constructor - https://html.spec.whatwg.org/multipage/comms.html#the-eventsource-interface